docs/.github/workflows/nightly-release.yml

```
@@ -0,0 +1,52 @@
+on:
+  schedule:
+    # Runs at 00:00 UTC every day
+    - cron: '0 0 * * *'
+  
+  # Allow manual trigger
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)